### ISSUE TITLE
Add applied data science workflow

### ISSUE THREAD

**Issue by markroepke** [uc-python/intro-python-datasci#25]
We should add a slide at the beginning of each section highlighting where we are in the applied data science workflow. This will help provide context to each section and how it can help students work with data.

**Comment by eswan18** [comment]
@bradleyboehmke Thoughts on this? Worth the effort?

**Comment by bradleyboehmke** [comment]
I don't think it would be a big effort. This is what I did with the Intro to R workshop where I just highlighted the part of the workflow that module focuses (i.e. [importing slide](https://uc-r.github.io/Intro-R/day-1b-import.html#2)).  

Adding the image is not a big deal. What is a larger question is whether we need to add a section that focuses more on "tidying". Right now we don't really have anything that focuses on that topic. We have importing, transforming (i.e. filtering, selecting, aggregation, joining), and visualization.  

Tidying focuses more on getting the data in the right shape (i.e. pivoting), taking care of missing values, removing duplicates, cleaning up column names (i.e. no spaces).  Here is what I covered in the R version of tidying [[link](https://uc-r.github.io/Intro-R/day-2c-tidy.html#1)].

**Comment by eswan18** [comment]
I'm not sure about the tidying bit. I suppose we could repurpose the time saved by simplifying importing and exporting (#33) to cover this. But does the flights data really lend itself to this? I'm not sure what examples we would have to cover.
